state-chain/runtime/src/migrations/arbitrum_integration.rs

```diff
@@ -138,6 +150,12 @@ impl OnRuntimeUpgrade for ArbitrumIntegration {
 		);
 		pallet_cf_environment::ArbitrumChainId::<Runtime>::put(chain_id);
 		pallet_cf_environment::ArbitrumSupportedAssets::<Runtime>::insert(ArbUsdc, usdc_address);
+		pallet_cf_chain_tracking::CurrentChainState::<Runtime, ArbitrumInstance>::put(ChainState {
+			block_height: start_block_number,
+			tracked_data: ArbitrumTrackedData {
+				base_fee: 0,
```

Comment:
after a single chain tracking operation, this will be filled, and ready for any fee related estimates, won't cause any trouble not setting this "correctly"